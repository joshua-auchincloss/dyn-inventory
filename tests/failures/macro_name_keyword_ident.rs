// using a reserved keyword as macro_name should not parse as Ident in this position
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    TraitC: StructC<T> {
        a: char,
        t: T,
    };
    macro_name = fn,
);

fn main() {}
