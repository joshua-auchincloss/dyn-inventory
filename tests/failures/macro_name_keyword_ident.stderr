error: expected identifier, found keyword `fn`
  --> tests/failures/macro_name_keyword_ident.rs:18:18
   |
18 |     macro_name = fn,
   |                  ^^
